bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -138,7 +139,7 @@ def AddressOf(self, obj):
         addr = self.addressof(instance=obj, byref=True)
 
         # Create a buffer (LowLevelView) from address
-        return CreateBufferFromAddress(addr)
+        return cppyy.ll.cast['std::intptr_t*'](addr)
```

Comment:
Note that in the original code, there was instead a preprocessor macro, selecting the right pointer type based on the architecture flag `R__B64`. So using `std::intptr_t` is a simplification here.

Also, I don't really understand why this should be a pointer to an integer and not `void*` the pointer points to some objects, so it makes not sense to cast to a pointer to an integer. But well, it's what the rest requires:
https://github.com/root-project/roottest/blob/master/python/cpp/PyROOT_cpptests.py#L374

So to keep the changes in this PR purely at the language translation level, I kept the type as before.